src/main/java/io/lumigo/core/utils/StringUtils.java

```diff
@@ -82,4 +83,10 @@ public static String buildMd5Hash(String s) {
     public static String dynamodbItemToHash(Map<String, AttributeValue> item) {
         return buildMd5Hash(ItemUtils.toItem(item).toJSON());
     }
+
+    public static int getBase64Size(String value) {
+        return (int)
+                Math.round(
+                        (Math.floor((value.getBytes(StandardCharsets.UTF_8).length / 3) + 1) * 4));
```

Comment:
Maybe add comments explaining the number 3, 1, 4. I don't understand why you did this.